### ISSUE TITLE
Considering moving mitogen under ansible-community organization

### ISSUE THREAD

**Issue by ssbarnea** [dw/mitogen#686]
As I recently moved several projects under https://github.com/ansible-community I wonder if it would not be a good idea to do the same about mitogen.

The main advantage of this would increase visibility with Ansible community and also ability to benefit from Zuul CI, which can also test changes across projects (testing if a specific change made on Ansible could fix a problem in mitogen).

**Comment by s1113950** [comment]
@dw  WDYT? I think it could be a good idea, especially when tests currently have issues: 
https://github.com/dw/mitogen/pull/693 `master` doesn't pass tests at the moment, which could be due to flaky/failing tests I found from running https://github.com/dw/mitogen/pull/658

One thing to consider though is mitogen is used for more than just Ansible 🤔 : https://mitogen.networkgenomics.com/

**Comment by ssbarnea** [comment]
I should mention that I am not part of the Ansible core or community team, but I doubt they would have any issues with that.

Due to @s1113950 comment I just realised that there are other reasons to be worried, changes being merged without running CI on them.

I know about mitogen for more than two years and tried it several times, still I was never able to pass the local experiment phase because I faced deal-breaker bugs. Maybe it would make sense to improve the CI coverage, even if this would involve marking some tests with xfail.

**Comment by s1113950** [comment]
I didn't mark them with xfail (not sure how to do that with the test framework being used), but I commented out a bunch of the bad ones and made Github tickets for them under the `ci` label. #658  now passes CI, and I'm waiting on people who have reported various interpreter discovery bugs to test my patch before it gets merged!
